packages/block-editor/src/utils/dom.js

```diff
@@ -11,6 +11,15 @@ export function getBlockDOMNode( clientId ) {
 	return document.querySelector( '[data-block="' + clientId + '"]' );
 }
 
+export function getClockPreviewContainerDOMNode( clientId ) {
```

Comment:
Is this a typo? Did you mean `getBlockPreviewContainerDOMNode` (note `Block` not `Clock`)? 😄

Reply:
I've fixed this.